runtime/j9vm/jvm.c

```diff
@@ -4474,11 +4504,18 @@ JVM_RaiseSignal(jint sigNum)
 {
 	jboolean rc = JNI_FALSE;
 	J9JavaVM *javaVM = (J9JavaVM *)BFUjavaVM;
+	BOOLEAN isShutdownSignal = isSignalUsedForShutdown(sigNum);
 
 	Trc_SC_RaiseSignal_Entry(sigNum);
 
-	if (J9_ARE_ALL_BITS_SET(javaVM->sigFlags, J9_SIG_XRS)
-			&& isSignalSpecial(sigNum)) {
+	if ((J9_ARE_ANY_BITS_SET(javaVM->sigFlags, J9_SIG_XRS | J9_SIG_XRS_ASYNC)
```

Comment:
-Xrs:sync is supposed to have the JVM continue to handle SIG_TERM, which is one of the shutdown signals.

Reply:
`-Xrs:sync` seems to be an OpenJ9 only option. should the bit-check above exclude the `J9_SIG_XRS_ASYNC` flag?

Reply:
it isn't `SIGTERM`. `SIGQUIT` should be allowed with `-Xrs:sync`. `JVM_RegisterSignal` shouldn't prevent the JVM to register a handler for `SIGQUIT` via `stdinit.c::standardInit -> J9SigQuitStartup`. `JVM_RegisterSignal` will only prevent a user to register a signal via `sun.misc.Signal.register`.

Reply:
https://www.ibm.com/support/knowledgecenter/en/SSYKE2_8.0.0/com.ibm.java.lnx.80.doc/diag/appendixes/cmdline/Xrs.html mentions both SIGQUIT and SIGTERM

Reply:
i account for `-Xrs`, `-Xrs:sync` and `-Xrs:async` in `JVM_RegisterSignal` and `JVM_RaiseSignal`.